src/redirect/config.js

```diff
@@ -0,0 +1,8 @@
+/* eslint-disable no-undef */
```

Comment:
We should probably set `process` as a [global variable](https://eslint.org/docs/user-guide/configuring/language-options#specifying-globals) in the eslint config, since it is a [global object in node](https://www.freecodecamp.org/news/node-process-object-explained/):

In `.eslintrc.js`:
```
  globals: {
    process: true
  },
```

This should allow you to remove `/* eslint-disable no-undef */` from the top of all of the files in this PR.